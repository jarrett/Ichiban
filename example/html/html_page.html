<h1>HTML Page</h1>

<p>This is a paragraph. 2 + 2 = <%= 2 + 2 %>. Current path: <%= current_path %></p>