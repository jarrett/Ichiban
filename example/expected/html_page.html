<!DOCTYPE html>
<html>
  <head>
    <title>Example Site</title>
  </head>
  
  <body>
    <h1>Example Site</h1>
    
    <h1>HTML Page</h1>

<p>This is a paragraph.</p>
  </body>
</html>